cabal.project

```diff
@@ -58,3 +58,8 @@ write-ghc-environment-files: always
 -- Do NOT add more source-repository-package stanzas here unless they are strictly
 -- temporary! Please read the section in CONTRIBUTING about updating dependencies.
 
+source-repository-package
```

Comment:
I don't think you need SRP for this change.

Reply:
Initially I wanted to show this was compatible with https://github.com/IntersectMBO/cardano-api/pull/635, but yeah you're right. Removed the SRP.